apps/src/music/views/Callouts.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+import {MusicState} from '../redux/musicRedux';
+import moduleStyles from './callouts.module.scss';
+const FontAwesome = require('../../templates/FontAwesome');
+
+const availableCallouts: {
+  [key: string]: {selector: string; arrowAngle: number};
+} = {
+  'play-sound-block': {
```

Comment:
Is the idea for this system that every time a levelbuilder wants a new callout we need to do a code update?

Reply:
At least for this initial implementation, yes.  We might be able to develop something a bit more flexible, but I do want to enforce a very clean interface between lab implementation and levelbuilder authoring.

Reply:
This does seem like the kind of thing that's going to be a source of frustration for levelbuilders in the long term, having to rely on the engineering team to be able to do new things in this space. It probably does make sense to keep it this simple for an initial implementation, but I'd love to see a comment here explaining some ways this could be iterated on in the future.

Reply:
+1, long term this will be frustrating for both levelbuilders and engineers.

Reply:
I do suspect there might be a middle ground in which we provide some parameters that scale to a broader range of desired targets.  For example, there might be a way to specify any type of Blockly block, without engineers needing to support each one individually.

But I do want us to be careful here.  We've gotten into a really bad situation with validations, in which there is a large amount of JS validation code inside levels, which hasn't been code reviewed, isn't easy to read in our repo, and which pokes deep inside our internal lab implementations.  I think we should take a bit of upfront inconvenience to ensure that we draw the right borders between systems.